WORKSPACE

```
@@ -153,64 +153,47 @@ llvm_zlib_system(name = "zlib")
 # Flex/Bison rules
 ###############################################################################
 
-rules_m4_version = "0.2.1"
+rules_m4_version = "0.2.3"
 
 http_archive(
     name = "rules_m4",
-    patch_args = ["-p1"],
-    patches = [
-        # Trying to upstream: https://github.com/jmillikin/rules_m4/pull/15
-        "@carbon//bazel/patches/m4:0001_Support_M4_building_on_FreeBSD.patch",
-    ],
-    sha256 = "eaa674cd84546038ecbcc49cdd346134a20961a41fa1a541e80d8bf4b470c34d",
-    strip_prefix = "rules_m4-{0}".format(rules_m4_version),
-    urls = ["https://github.com/jmillikin/rules_m4/archive/v{0}.tar.gz".format(rules_m4_version)],
+    sha256 = "10ce41f150ccfbfddc9d2394ee680eb984dc8a3dfea613afd013cfb22ea7445c",
+    urls = ["https://github.com/jmillikin/rules_m4/releases/download/v{0}/rules_m4-v{0}.tar.xz".format(rules_m4_version)],
 )
 
 load("@rules_m4//m4:m4.bzl", "m4_register_toolchains")
 
 # When building M4, disable all compiler warnings as we can't realistically fix
 # them anyways.
-m4_register_toolchains(extra_copts = ["-w"])
+m4_register_toolchains(version = "1.4.18", extra_copts = ["-w"])
```

Looking at the code, it seems to still support omitting the `version` here and picks the default one?

```suggestion
m4_register_toolchains(extra_copts = ["-w"])
```